### PR TITLE
feat(perf): improvements

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

File: jest.config.unit.ts [hunk 1]
```diff
@@ -13,8 +13,7 @@ export default {
   preset: 'ts-jest',
   testEnvironment: 'jest-environment-node-single-context',
   moduleFileExtensions: ['js', 'ts'],
-  testMatch: ['<rootDir>/test/unit/**/?(*.)+(spec|test).(ts|js)'],
-  testPathIgnorePatterns: ['dist/', 'node_modules/'],
+  testMatch: ['<rootDir>/src/**/?(*.)+(spec|test).[jt]s?(x)'],
```
Comment (ChaituVR) [inline jest.config.unit.ts:16]: Are we moving this in all repos?

**Review by wa0x6e — APPROVED** [review]
utAck

**Comment by wa0x6e** [comment]
May not be in the scope of this PR, but what about moving the cache layer to the express router level ? So we could handle it with a middleware

**Review by ChaituVR — APPROVED** [review]
utAck

**Pull request merged**